### PR TITLE
Hero Changes

### PR DESCRIPTION
**Pull request by Tronicality**
Added Shade Vengeance
Candy Sugar Rush no longer adds speed
Magmax Cooldown changed from 0.5 to 0.25

### CODE REVIEW

File: index.html
Comment (Tronicality) [inline index.html:1]: Honestly i just removed a space, I don't know why it says I changed the whole file

File: entities.js [hunk 1]
```diff
@@ -1069,19 +1075,197 @@ class Shade extends Player {
...
+      this.secondAbilityActivated = !this.secondAbilityActivated;
+      this.secondAbilityCooldown+=1000;
+      this.energy-=5;
+
+      area.addEntity(0, new shadeVengeance(new Vector(this.pos.x - offset.x, this.pos.y - offset.y),this.mouseActive ? this.mouse_angle : this.previousAngle, 1.3, 58, this.id))
+    }
+  }
+}
+
+class shadeVengeance extends Entity {
```
Comment (sonic3XE) [inline entities.js:1098]: :o

Comment (sonic3XE) [reply entities.js:1098]: @Ino5559 take a look at this code.

**Comment by Ino5559** [comment]
so if i use my brain to this nice code but who this hero


вт, 11 июн. 2024 г. в 23:56, sonic3XE ***@***.***>:

**Comment by Pifary-dev** [comment]
Oops, I forgot about this. Thanks for the update

**Pull request merged**